src/lib/operators/import.cpp

```diff
@@ -51,9 +53,37 @@ std::shared_ptr<const Table> Import::_on_execute() {
   auto table = std::shared_ptr<Table>{};
 
   switch (_file_type) {
-    case FileType::Csv:
-      table = CsvParser::parse(filename, _chunk_size, _csv_meta);
+    case FileType::Csv: {
+      if (static_cast<int>(static_cast<bool>(_csv_meta)) +
```

Comment:
I'm not sure how the others see this, but I'd prefer a priority ranking that chooses the first available definition source instead of failing if multiple are there.
I like the order determined by the if statement below.

Reply:
Failing if multiple are there was Martins proposal. His argument was that it is unintuitive if there is a hidden ranking somewhere, and that we should rather fail if two sources are given.

Reply:
Hm, sounds reasonable.
Let's keep it that way :)